PasscodeLock/PasscodeLock/PasscodeLock.swift

```diff
@@ -12,7 +12,7 @@ import LocalAuthentication
 open class PasscodeLock: PasscodeLockType {
     
     open weak var delegate: PasscodeLockTypeDelegate?
-    open let configuration: PasscodeLockConfigurationType
+    public let configuration: PasscodeLockConfigurationType
```

Comment:
As far as I can see we don't override this in our code. So that should be OK, but I'm wondering why the change of this access modifier - and also the one in `PasscodeLockPresenter` - is part of a swift migration. Was it done by the automatic migration or was this a manual change?

Reply:
Automatic. I tried to undo it, but got a compiler error then.